roles/openshift_aws/defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
+openshift_aws_build_ami_ssh_user: centos
```

Comment:
I'm not sure centos should be the default. This references the aws ssh key to build an ami, maybe amibuilder?

Reply:
You're right; I don't believe this should be defined here, I will remove it.